workflows/gpu_operator_versions/update_versions.py

```
@@ -9,46 +9,164 @@
+                f'OpenShift version "{ocp_version}" is not in the list of releases. '
+                f'Check if there was an update to an old version.'
+            )
+            continue
+
+        ocp_config = get_ocp_support_config(ocp_version, support_matrix)
+
+        if ocp_config.get(CONFIG_STATUS) == STATUS_MAINTENANCE:
+            # Maintenance versions: test only with pinned GPU operators
+            pinned_gpus = normalize_pinned_gpu_operator(ocp_config.get(CONFIG_PINNED_GPU_OPERATOR))
+            for pinned_gpu in pinned_gpus:
+                if not pinned_gpu in gpu_releases:
```

Ruff (0.14.3)

Test for membership should be `not in`

Convert to `not in`

(E713)